templates/base.html

```diff
@@ -142,6 +142,17 @@ <h2 class="white-color">Login</h2>
             </button>
         </div>
 
+        <!-- Google Analytics -->
```

Comment:
I would move this to be after all of our other vendor JS, so after the line that says:
```html
<script type="text/javascript">$(document).foundation();</script>
```

Additionally, I think all of the JS in this script should indented in, and the open `<script>` tag should explicitly state `type="text/javascript"` like the other tags.